doc/manual/cookbooks/initial-condition-S20RTS/S20RTS.input.prm

```diff
@@ -0,0 +1,5 @@
+             20 111111111111111111111  24 000111111111111111111111
```

Comment:
it isn't a .prm file. just rename the file to S20RTS.input and remove the .prm.out file, then adjust the reference in manual.tex.